packages/talker_flutter/lib/src/ui/widgets/snackbar.dart

```diff
@@ -5,19 +5,27 @@ class SnackbarContent extends StatelessWidget {
     Key? key,
     required this.message,
     required this.title,
-    this.bacgroundColor = Colors.red,
+    this.backgroundColor = Colors.red,
+    this.dismissButton,
+    this.titleTextStyle,
+    this.messageTextStyle,
+    this.buttonDismissText,
   }) : super(key: key);
 
   final String title;
   final String message;
-  final Color bacgroundColor;
+  final String? buttonDismissText;
+  final Color backgroundColor;
+  final Widget? dismissButton;
```

Comment:
Maybe it 's better to call this field **dismissButtonText** like **dismissButton**?